angular/projects/admin-nrpti/src/app/records/administrative-sanctions/administrative-sanction-add-edit/administrative-sanction-add-edit.component.ts

```diff
@@ -97,6 +101,32 @@ export class AdministrativeSanctionAddEditComponent implements OnInit, OnDestroy
     }
   }
 
+  private subscribeToFormControlChanges() {
+    // listen to legislation control changes
+    const debouncedUpdateLegislationDescription = this.utils.debounced(500, () => this.updateLegislationDescription());
+    this.myForm
+      .get('legislation')
+      .valueChanges.pipe(takeUntil(this.ngUnsubscribe))
+      .subscribe(c => {
```

Comment:
c isn't used anywhere

Reply:
yeah, and also why its just 'c'.  Must be leftover from testing something.  Will remove.

Reply:
Fixed